styles/onyx/components/modal/_variables.scss

```diff
@@ -1,10 +1,10 @@
-@import '@addepar/style-toolbox/core-variables/colors';
-@import '@addepar/style-toolbox/core-variables/spacing';
-@import '@addepar/style-toolbox/core-variables/typography';
-@import '@addepar/style-toolbox/utilities/typography/mixins';
-@import '@addepar/style-toolbox/utilities/typography/fonts';
-@import '@addepar/style-toolbox/core-variables/borders';
-@import '@addepar/style-toolbox/elements/inputs/button/index';
+@import '@addepar/style-toolbox/onyx/core-variables/colors';
```

Comment:
While we're in here changing the urls...
Lets change them all to use relative paths (ditto for the other components). It worked for the Ember components bc of magic, but doesn't work for other projects. >.<
Relative should work for all though.